website/docs/result_to_dict.md

```diff
@@ -0,0 +1,127 @@
+---
+id: result_to_dict
+title: Result to Dictionary
+sidebar_label: Result to Dictionary
+---
+## result_to_dict() Function
+Will format any value a function returns.
+For e.g., will return tuple as a tuple, dict as a dict
+
+Datatypes supported:
+
+SQLAlchemy
+ - Varchar
+ - ENUM
+ - Integer
+ - Float
+ - Text
+ - JSON ( converted to `dict` )
+ - BLOB/Binary
+ - Boolean
+ - Unicode
+ - Date ( converted to `timestamp` )
+ - Datetime ( converted to `timestamp` )
+
+Python
+ - Point (Geographical point)
+ - Float
+ - Tuple
+ - List
+ - Dictionary
+ - Set
+ - BLOB/Binary

```

Comment:
`BLOB/Binary` is python type?

Reply:
Binary is a python type, I'll remove the `BLOB`